services/src/main/java/org/keycloak/services/managers/ResourceAdminManager.java

```diff
@@ -197,14 +198,16 @@ public static String getBackchannelLogoutUrl(KeycloakSession session, ClientMode
         return StringPropertyReplacer.replaceProperties(absoluteURI);
     }
 
-    protected Response sendBackChannelLogoutRequestToClientUri(ClientModel resource,
+    protected Response sendBackChannelLogoutRequestToClientUri(ClientModel client,
                                                               AuthenticatedClientSessionModel clientSessionModel, String managementUrl) {
         UserModel user = clientSessionModel.getUserSession().getUser();
 
-        LogoutToken logoutToken = session.tokens().initLogoutToken(resource, user, clientSessionModel);
+        LogoutToken logoutToken = session.tokens().initLogoutToken(client, user, clientSessionModel);
+        logoutToken = new TokenManager().transformLogoutToken(session, client, logoutToken);
```

Comment:
Minor: Maybe the call to "transformLogoutToken" can be added directly to the `session.tokens().initLogoutToken` method, so there is no need to call additional things from ResourceAdminManager?

Reply:
Yes this sounds like a good idea, I'll move the call into initLogoutToken.